### PR TITLE
Update config.yml

### PR DESCRIPTION
**Pull request by faithngetich**
Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

Fixes https://github.com/publiclab/plots2/issues/3681

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] PR body includes `fixes #0000`-style reference to original issue #
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

**Comment by DivyeshPuri** [comment]
@faithngetich Can you please reference the issue number corresponding to this PR so that we can have a look at what issue you are trying to solve.

**Comment by faithngetich** [comment]
@DivyeshPuri Issue [link](https://github.com/publiclab/plots2/issues/3681#issuecomment-429254015)

File: .github/config.yml [hunk 1]
```diff
@@ -3,19 +3,23 @@
...
+ Thanks for opening this pull request! `Dangerbot` will test out your code and reply in a bit with some pointers and requests.
+ There may be some errors, **but don't worry!** We're here to help! 👍🎉😄
 # Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
 
 # Comment to be posted to on pull requests merged by a first time user
-firstPRMergeComment: >
-  Congrats on merging your first pull request! 🙌🎉⚡️
+firstPRMergeComment: |
+ Congrats on merging your first pull request! 🙌🎉⚡️ 
+ Your code will likely be published to https://code.publiclab.org in the next few days.
```
Comment (jywarren) [inline .github/config.yml]: Hi! Here we may want to keep plain publiclab.org as that's where the code will be published. Would you mind making this change? Thanks!!!!

**Comment by faithngetich** [comment]
@jywarren I've made the changes, let me know if my change is as expected.

**Comment by jywarren** [comment]
Ah, i'm sorry - could we actually add back in the `https://` because i don't believe it will appear as a link otherwise! Thank you

**Comment by faithngetich** [comment]
No worries @jywarren , I have added.

**Pull request merged**